client/src/Signup/Confirmation.js

```diff
@@ -1,9 +1,57 @@
 import React from 'react'
+import PropTypes from 'prop-types'
+import { Divider, Typography } from 'antd'
+import LabelImportantIcon from '@material-ui/icons/LabelImportant'
 
-export default function Confirmation() {
+const { Title, Text, Link } = Typography
+
+const userEmail = 'chelsea@pieforproviders.com'
```

Comment:
Thoughts on moving these to heroku config variables? @katelovescode

Reply:
I could totally see that for the `pieEmail`. I was thinking 'userEmail` is just a placeholder for the actual user that had signed up's email. I'll add a comment to make that clear!